style.css

```diff
@@ -5,76 +5,339 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
 
+.header {
+  display: flex;
+  align-items: center;
+  padding: 1.2rem;
+  margin-left: 0.5rem;
+  justify-content: space-between;
+}
+
+.toolBar {
+  color: #6070ff;
+  font-size: 1.5rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+.firstSec {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+  width: 100%;
+}
```

Comment:
- [ ] For your first section you should input background image to align given figma design and if you input the image your design is better look good.

**Your Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/c1fcdd65-56b1-43b2-9c9b-64e1bb73caa9)
 
**Figma Design**

![image](https://github.com/mioOleen/Portfolio-1/assets/118582316/05581a3c-795b-4c44-bfdd-619b63d1af15)